feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -264,67 +247,73 @@ func testIPv4BackUpSwitch(ctx context.Context, t *testing.T, args *testArgs) {
 	// validate programming using AFT
 	aftCheck(t, args.dut, dstPfx, []string{"192.0.2.6", "192.0.2.10"})
 	// Validate traffic over primary path port2, port3
-	validateTrafficFlows(t, args.ate, BaseFlow, false, []string{"port2", "port3"})
+	validateTrafficFlows(t, args.ate, args.top, BaseFlow, false)
 
 	//shutdown port2
-	flapinterface(t, args.ate, "port2", false)
-	defer flapinterface(t, args.ate, "port2", true)
+	flapinterface(t, args.dut, "port2", false)
+	defer flapinterface(t, args.dut, "port2", true)
 	// validate programming using AFT
 	aftCheck(t, args.dut, dstPfx, []string{"192.0.2.10"})
 	// Validate traffic over primary path port3
-	validateTrafficFlows(t, args.ate, BaseFlow, false, []string{"port3"})
+	validateTrafficFlows(t, args.ate, args.top, BaseFlow, false)
 
 	//shutdown port3
-	flapinterface(t, args.ate, "port3", false)
-	defer flapinterface(t, args.ate, "port3", true)
+	flapinterface(t, args.dut, "port3", false)
+	defer flapinterface(t, args.dut, "port3", true)
 	// validate programming using AFT
 	aftCheck(t, args.dut, dstPfx, []string{"192.0.2.14"})
 	// validate traffic over backup
-	validateTrafficFlows(t, args.ate, BaseFlow, false, []string{"port4"})
+	validateTrafficFlows(t, args.ate, args.top, BaseFlow, false)
 }
 
 // createFlow returns a flow from atePort1 to the dstPfx
-func createFlow(t *testing.T, ate *ondatra.ATEDevice, top *ondatra.ATETopology, name string) *ondatra.Flow {
-	srcEndPoint := top.Interfaces()[atePort1.Name]
-	dstEndPoint := []ondatra.Endpoint{}
-	for intf, intf_data := range top.Interfaces() {
-		if intf != "atePort1" {
-			dstEndPoint = append(dstEndPoint, intf_data)
-		}
-	}
-	hdr := ondatra.NewIPv4Header()
-	hdr.WithSrcAddress(dutPort1.IPv4).DstAddressRange().WithMin(dstPfxMin).WithCount(1)
-
-	flow := ate.Traffic().NewFlow(name).
-		WithSrcEndpoints(srcEndPoint).
-		WithDstEndpoints(dstEndPoint...).
-		WithHeaders(ondatra.NewEthernetHeader(), hdr).WithFrameSize(300)
+func createFlow(t *testing.T, ate *ondatra.ATEDevice, top gosnappi.Config, name string) string {
+
+	flow := top.Flows().Add().SetName(name)
+	flow.Metrics().SetEnable(true)
+	e1 := flow.Packet().Add().Ethernet()
+	e1.Src().SetValue(atePort1.MAC)
+	flow.TxRx().Port().SetTxName(atePort1.Name)
+	waitOTGARPEntry(t)
+	dstMac := ate.OTG().Telemetry().Interface(atePort1.Name + ".Eth").Ipv4Neighbor(dutPort1.IPv4).LinkLayerAddress().Get(t)
+	e1.Dst().SetChoice("value").SetValue(dstMac)
+	v4 := flow.Packet().Add().Ipv4()
+	v4.Src().SetValue(atePort1.IPv4)
+	v4.Dst().Increment().SetStart(dstPfxMin).SetCount(1)
+	ate.OTG().PushConfig(t, top)
+	ate.OTG().StartProtocols(t)
```

Comment:
Is this needed?

Reply:
After every push config we need to include the Start Protocols in order to have this test working fine on hw

Reply:
Couple questions:
- Is that specific to hardware? (If so, worth a comment.)
- What specifically happens with protocol state in this case? (Eg. it seems we're not restarting everything since we're not waiting for the ARP entry again)

Reply:
After every PushConfig, StartProtocols is required to run the test on hardware. There without the protocols (control plane) started, we can't apply/start the traffic items where ARP has to be resolved.